### ISSUE TITLE
Add missing submit button to form

### ISSUE THREAD

**Issue by wesley-dean-gsa** [GSA/usagov-benefits-eligibility#421]
# User Story

As a BEARS **user**, in order to **submit the form without a keyboard**, I would like **a submit button added to the form** .

Pa11y reports that there's a form on the death-of-a-loved-one page that does not have a submit button.  Looking at the site, I think it's referring to the from on the left side of the page ("start here" and "Your date of birth is"...).

I understand that interacting with elements of the form trigger events and that a submit button may not be strictly necessary in order for one to interact meaningfully with the site.  I haven't looked to see what would happen if that form was submitted.

Therefore, it may be the case that we can document that this finding is a non-issue and mark this issue as `won't fix` .

A sample Pa11y report follows:

```
 > usagov-benefits-eligibility@0.0.1 test:pa11y > pa11y-ci
Running Pa11y on 1 URLs:

http://localhost:3000/death-of-a-loved-one/? - 1 errors

Errors in http://localhost:3000/death-of-a-loved-one/?:

• This form does not contain a submit button, which creates issues for those
who cannot submit the form using the keyboard. Submit buttons are INPUT
elements with type attribute "submit" or "image", or BUTTON elements with
type "submit" or omitted/invalid.

(#main-content > div > section > div:nth-child(3) > div:nth-child(1) > div >
section > div > div > div > form)
```

Acceptance Criteria:

- [ ] Accessibility concerns are resolved or documented
- [x] Pa11y no longer generates this finding

Definition of Done:

- [x] Code complete
- [x] Tests coverage is greater than team benchmark (90% goal)
- [x] Security scans passed
- [x] Acceptance Criteria is met and it works as expected
- [ ] Accessibility tested
- [ ] Build process and deployment is automated and repeatable
- [ ] Load testing/performance testing
- [ ] Self Documentation whenever possible
- [ ] Feature toggles if appropriate
- [ ] Deployed to staging
- [ ] Usability testing
- [ ] PR approved / Peer reviewed
- [ ] PO approved

**Comment by lgarceau768** [comment]
Fixed in PR #432

**Comment by lgarceau768** [comment]
Closed due to verification of completion.
